config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -944,42 +927,36 @@ def generate_network_plugins():
 ################################
 def generate_upgrades():
 
-    kops24 = 'v1.24.5'
     kops25 = 'v1.25.4'
     kops26 = 'v1.26.4'
-    kops27 = 'v1.26.7-beta.3'
+    kops27 = 'v1.27.0-beta.3'
 
     versions_list = [
         #  kops    k8s          kops      k8s
-        # 1.25 release branch
-        ((kops24, 'v1.19.16'), ('1.25', 'v1.20.6')),
-        ((kops24, 'v1.23.1'), ('1.25', 'v1.23.1')),
-        ((kops25, 'v1.21.14'), ('1.25', 'v1.22.13')),
-        ((kops25, 'v1.25.0'), ('1.25', 'v1.25.0')),
         # 1.26 release branch
-        ((kops24, 'v1.20.6'), ('1.26', 'v1.21.7')),
-        ((kops24, 'v1.24.0'), ('1.26', 'v1.25.0')),
         ((kops25, 'v1.20.6'), ('1.26', 'v1.21.7')),
         ((kops25, 'v1.25.0'), ('1.26', 'v1.26.0')),
         ((kops26, 'v1.21.14'), ('1.26', 'v1.22.13')),
         ((kops26, 'v1.26.0'), ('1.26', 'v1.26.0')),
         # 1.27 release branch
-        ((kops25, 'v1.20.15'), ('1.27', 'v1.21.14')),
+        ((kops25, 'v1.20.15'), ('1.27', 'v1.22.17')),
```

Comment:
Does k8s 1.22 support kubelet 1.20?

Reply:
Not sure I get the Q

Reply:
It is an experiment.

Reply:
Ah, the [version skew policy](https://kubernetes.io/releases/version-skew-policy/) still states that kube-apiserver only supports a skew of one version. So an HA cluster can only upgrade one k8s version at a time. The supported version skew between kubelet and kube-apiserver is two, so a single-control-plane-node cluster could theoretically upgrade two k8s versions, though I don't think this is worth testing.

If it is an experiment, it should be commented as such so we don't keep propagating it forward.

Reply:
Removed